flutter-idea/src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -156,6 +156,15 @@ private String getRunScriptFromWorkspace(@NotNull final Project project) {
     return runScript;
   }
 
+  private String getToolsScriptFromWorkspace(@NotNull final Project project) {
+    final Workspace workspace = getWorkspace(project);
+    String toolsScript = workspace == null ? null : workspace.getToolsScript();
+    if (toolsScript != null) {
+      toolsScript = workspace.getRoot().getPath() + "/" + toolsScript;
```

Comment:
Does this work on Windows? I think this could happen.

Reply:
I can use a separator char instead here